include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -973,6 +973,16 @@ struct OrtApi {
    * This is a no-copy method whose pointer is only valid until the backing OrtValue is free'd.
    */
   ORT_API2_STATUS(TensorAt, _Inout_ OrtValue* value, size_t* location_values, size_t location_values_count, _Outptr_ void** out);
+
+  /*
+    * Get count of custom op domains
+    */
+  ORT_API2_STATUS(GetCustomOpDomainsCount, _Inout_ OrtSessionOptions* options, _Out_ size_t* out);
```

Comment:
I don't think we need these 2 APIs. The user who creates the OrtCustomOpDomain obj using the CreateCustomOpDomain API is already in possession of the corresponding pointer and can arrange to release it.

Reply:
sure